docs/virtual_machines/usb_host_passthrough.md

```diff
@@ -0,0 +1,86 @@
+# USB Host Passthrough
+
+This feature is another case of [Host Devices Assigment](host-devices.md), where we can provide a
+USB devices that is plugged in a Node to the VM running in the same Node.
+
+## Requirements
+
+Cluster admin privilege to edit the KubeVirt CR in order to
+- [Enable feature gate](../operations/activating_feature_gates.md) HostDevices
+- Edit the `permittedHostDevices` configuration to expose node USB devices to the cluster
+
+## Exposing USB Devices
+
+In order to assign USB devices to your VMI, you'll need to expose those devices to the cluster under
+a resource name. For this example, we will use the `kubevirt.io/storage` resource name for the
+device with `vendor: "46f4"` and `product: "0001"` [^1].
```

Comment:
Re: the footnote. We will have to enable footnotes in the mkdocs.yml (https://squidfunk.github.io/mkdocs-material/reference/footnotes/). We can do that in this PR or we can create a secondary one just for that.
Alternatively it could become a note admonition.

Reply:
Either way is fine for me.

Reply:
https://github.com/kubevirt/user-guide/pull/723 is merged

Reply:
Thanks, resolving this one.